### PR TITLE
Added rules to dangerfile

### PR DESCRIPTION
**Pull request by l2succes**
Making sure the newly added PR danger rules work correctly.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please add a description to your PR.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
Awesome, wanna also update Danger? Then instead of 

screen shot 2017-01-24 at 1 46 25 pm

we get

screen shot 2017-01-24 at 1 46 37 pm

which is a littler bit better 👍

File: dangerfile.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* @flow */
 
-import { danger, fail } from 'danger'
+// Removed import
```
Comment (orta) [inline dangerfile.js]: this will need fixing for merging, its why CI is red

Comment (l2succes) [inline dangerfile.js]: done, should be green soon

**Comment by orta** [comment]
Awesome 👍

**Pull request merged**